src/Microsoft.DotNet.Build.Tasks/PackageFiles/scripts/docker/init-docker.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+set -e
+
+scriptName=$0
+
+function usage {
+    echo "Usage: $scriptName image"
+    echo "  image      Name of the Docker image to pull"
+
+    exit 1
+}
+
+# Executes a command and retry up to 5 times until if it fails.
+function execute {
+  local retries=5
+  local waitFactor=3
```

Comment:
can we provide options for changing the retries / wait factor with defaults as provided?